zeroex/orderwatch/expirationwatch.go

```diff
@@ -134,7 +134,7 @@ func (e *ExpirationWatcher) Receive() <-chan []ExpiredOrder {
 // to the caller
 func (e *ExpirationWatcher) prune() []ExpiredOrder {
 	pruned := []ExpiredOrder{}
-	currentTimestamp := time.Now().Unix()
+	currentTimestamp := time.Now().UTC().Unix()
```

Comment:
[`Time.Unix`](https://golang.org/pkg/time/#Time.Unix) already converts to UTC so you don't have to do this.

> Unix returns t as a Unix time, the number of seconds elapsed since January 1, 1970 UTC. The result does not depend on the location associated with t.